### DISCUSSION TITLE
Missing OffsetX and OffsetY in Touch spec

### DISCUSSION THREAD

**Discussion by Inspirateur** [mdn/mdn-community#449] [Content] [answered]
Hi, Mouse events provide us with various information about the mouse position including the very useful "OffsetX" and "OffsetY", the mouse position relative to the DOM element that emitted the event. 
Similarly, Touch events provide us much the same information, except for OffsetX and OffsetY !
(https://developer.mozilla.org/en-US/docs/Web/API/Touch)

I don't see any reason for these 2 to be omitted, and the fact that they are missing from the MDN Doc is actively blocking its implementation in frameworks such as dotnet (https://github.com/dotnet/aspnetcore/issues/41619#issuecomment-1122994392)

I think they should either be added, or a good reason for their omission be displayed in the Touch documentation :)

**Comment by yin1999** [comment, marked as answer]
Hey @Inspirateur. The `offsetX` and `offsetY` attributes are not listed in the [spec](https://w3c.github.io/touch-events/#touch-interface). So we will not document them. MDN will document features that have been implemented by browsers or be listed in specifications. If you want MDN to document these attributes, perhaps you should check or open an issue in the [spec repo](https://github.com/w3c/touch-events).

**Reply by Inspirateur** [reply]
ah my bad i was wondering if it was the right place and it was not 😬

thanks for pointing it to me!

**Reply by yin1999** [reply]
Never mind. I'm glad to be able to help you :smiley:

**Comment by yarusome** [comment]
See also: https://github.com/w3c/touch-events/issues/62.
